YouTubeCrawler/src/it/scala/com/sofiia/production/functions/YTVideoCrawlerTest.scala

```diff
@@ -0,0 +1,17 @@
+package com.sofiia.production.functions
+
+import java.io.FileInputStream
+import java.util.Properties
+
+import com.sofiia.production.functions.io.FileReader
+
+class YTVideoCrawlerTest extends YTTest {
+  test("YTVideoCrawler.videoItemsAreCrawled"){
+    val properties: Properties = new Properties()
+    properties.load(new FileInputStream(fileName))
+    val mostPopularVideosFileName = properties.getProperty("mostPopularVideoFile")
+    val file = FileReader.read(mostPopularVideosFileName)
+    val videoList = new YTVideoListParser().fromJson(file)
+    assert(videoList.nonEmpty)
```

Comment:
Let's make this check based on items count comparison. In your case, "at least once" logic will work but it's not 100% correct